Routing/Loader/Reader/RestActionReader.php

```diff
@@ -333,39 +333,41 @@ public function read(RestRouteCollection $collection, \ReflectionMethod $method,
 
             // add route to collection
             $route = new Route(
-                $path, $defaults, $requirements, $options, $host, [], $methods, $condition
+                $path, $defaults, $requirements, $options, $host, [], $methods, $versionCondition
```

Comment:
I'm not sure we should rename this variable, this looks a bit confusing with the one which keeps the same name.